examples/wiznet5k_simpleserver.py

```diff
@@ -31,6 +31,8 @@
 while True:
     conn, addr = server.accept()  # Wait for a connection from a client.
     with conn:
-        data = conn.recv()
-        print(data)
-        conn.send(data)  # Echo message back to client
+        while True:
+            data = conn.recv() 
```

Comment:
Specify `1024` bytes

Reply:
i'm not sure, how do i apply this? you mean, change the demo code like 1024 byte loopback (1024 Byte receive and then 1024 byte send)?

Reply:
`conn.recv(1024)`

Reply:
ok. changed it.